rules/app.bzl

```diff
@@ -12,6 +13,36 @@ _IOS_APPLICATION_KWARGS = [
     "launch_storyboard",
 ]
 
+def write_info_plists_if_needed(name, plists):
```

Comment:
I know this precedes this change, but when do we expect to see more than 1 plist supplied?

Reply:
`rules_apple` allows passing multiple plists and it coalesces them, so we're just supporting the same interface as the apple rules for consistency